Reports/progress_report.md

```diff
@@ -0,0 +1,295 @@
+- Drop Microsoft Remote Desktop: (PR: [#1159](https://github.com/alan-turing-institute/data-safe-haven/issues/1159) primarily for increased security as it shows more issues than Guacamole, in doing this several other open issues are resolved.
+- Documentation improvements and updates
+
+### Identify and implement core IAC changes: [#28](https://github.com/alan-turing-institute/data-safe-haven-team/issues/28)
+
+#### Goal
+Make DSH deployment more robust and development easier through using IAC and configuration management.
+- Take advantage of IAC and configuration management in the DSH codebase which will
+  - Make deployments faster
+  - Make deployments more reliable
+  - Make development easier
+- Move away from non-idempotent, bespoke scripts (Powershell, bash, cloud-init)
+
+##### Definition of done
+On the release of a new major version which removes legacy, script-based deployment.
+
+#### Progress
+Arrived at a IAC MVP version of the code, available as a penetration tested [pre-release](https://github.com/alan-turing-institute/data-safe-haven/releases/tag/v5.0.0-rc.1).
+
+This new code is better and easier for users to deploy, however some incompatibilities with the old code would require extensive work. 
+As PowerShell heads to its final release it was decided not to work on fixing these.
+
+Since finishing migration a lot of the work is focusing on structuring the code, small improvements aimed at the user experience and robustness.
+
+These are main references and milestones, a more complete list is available on the [story issue](https://github.com/alan-turing-institute/data-safe-haven-team/issues/28)
+- Codebase pre-release https://github.com/alan-turing-institute/data-safe-haven/releases/tag/v5.0.0-rc.1
+- Next version milestone https://github.com/alan-turing-institute/data-safe-haven/milestone/20
+
+## Information governance & standards
+Infrastructure adhering to the latest agreed upon standard
+Identifying, co-creating and supporting a TRE standard used across TRE infrastructures
+
+### Co-create a TRE standard (SATRE): [#23](https://github.com/alan-turing-institute/data-safe-haven-team/issues/23)
+
+Contributes to:
+- Identifying, co-creating and supporting a TRE standard used across TRE infrastructures
+
+#### Goal
+Develop the SATRE specification that UK TREs can evaluate themselves against
+
+##### Definition of done
+SATRE specification published
+
+#### Progress
+[SATRE specification](https://satre-specification.readthedocs.io/en/stable/) published, and available for contribution and reproducibility in its [open repository](https://github.com/sa-tre/satre-specification).
+
+Currently Turing and HIC have self-evaluated against it and evaluations are available openly, several conversations ongoing about other institutions doing the same and making them available.
```

Comment:
If we'd like to list orgs we know are going through this:
- KCL
- Sheffield
- North West NHS SNSDE
- NHS SDE technology group reviewing SATRE